app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,18 @@
   
   <%= yield %>
   
+<script src="https://www.gstatic.com/firebasejs/3.6.4/firebase.js"></script>
+<script>
+  // Initialize Firebase
+  var config = {
+    apiKey: "AIzaSyDcZeyaMD_zB8BKh8P1ciNhNwrPaFW9lqw",
```

Comment:
Oh nvm this was a Firebase key - I thought it was pusher. These are meant to be public, so this was fine.